cmd/imports/keys.go

```diff
@@ -0,0 +1,87 @@
+package imports
+
+import (
+	"context"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	"github.com/consensys/quorum-key-manager/pkg/json"
+	"github.com/consensys/quorum-key-manager/src/infra/log"
+	manifest "github.com/consensys/quorum-key-manager/src/infra/manifests/entities"
+	"github.com/consensys/quorum-key-manager/src/stores"
+	"github.com/consensys/quorum-key-manager/src/stores/database"
+	"github.com/consensys/quorum-key-manager/src/stores/entities"
+	"github.com/consensys/quorum-key-manager/src/stores/manager/keys"
+)
+
+func ImportKeys(ctx context.Context, db database.Keys, mnf *manifest.Manifest, logger log.Logger) error {
+	logger.Info("importing keys...", "store", mnf.Kind, "store_name", mnf.Name)
+
+	store, err := getKeyStore(mnf, logger)
+	if err != nil {
+		return err
+	}
+
+	storeIDs, err := store.List(ctx, 0, 0)
+	if err != nil {
+		return err
+	}
+
+	dbIDs, err := db.SearchIDs(ctx, false, 0, 0)
+	if err != nil {
+		return err
+	}
+
+	var n uint
+	for _, id := range difference(storeIDs, dbIDs) {
```

Comment:
This is needed because Ethereum accounts IDs are addresses and what you retrieve is an ID so it is impossible to have a list of addresses from the store that you compare with a list of addresses from DB. That is possible only for keys and secrets